src/java/org/apache/cassandra/index/sai/disk/v1/postings/PostingListRangeIterator.java

```diff
@@ -89,7 +89,7 @@ public PostingListRangeIterator(IndexIdentifier indexIdentifier,
     @Override
     protected void performSkipTo(PrimaryKey nextKey)
     {
-        if (skipToKey != null && skipToKey.compareTo(nextKey) > 0)
+        if (skipToKey != null && skipToKey.compareTo(nextKey, false) > 0)
```

Comment:
Note: This is functionally equivalent to what was here before, as the default `compareTo()` used to be "non-strict".